crates/sui-adapter-transactional-tests/tests/dynamic_fields/wrap_object.exp

```diff
@@ -14,26 +14,30 @@ mutated: object(0,0)
 gas summary: computation_cost: 1000000, storage_cost: 5890000,  storage_rebate: 0, non_refundable_storage_fee: 0
 
 task 3 'view-object'. lines 45-45:
-Owner: Account Address ( A )
+Owner: Object ID: ( fake(2,2) )
 Version: 2
 Contents: a::m::Obj {id: sui::object::UID {id: sui::object::ID {bytes: fake(2,1)}}}
 
 task 4 'run'. lines 47-47:
-Error: Object 0x519c6f4caae4a1a72604cbab9fd1847061b4011003634cb4cae02ae56d8e9f83 is owned by object 0xeaa96af86250771b379418b685c426f34cea4758d7c3db4534f323faa343eb36. Objects owned by other objects cannot be used as input arguments.
```

Comment:
Similar question here -- just want to draw attention to these tests because this is a behavioural change -- is this intended? I'm leaning towards yes (although then I'm confused how the test ended up with its old output).